src/components/Flyout/Flyout.tsx

```diff
@@ -1,35 +1,24 @@
+'use client';
+
 import styles from './Flyout.module.scss';
 import Button from '@components/Button/Button.tsx';
 import { useCardStore } from '@/core/store/useCardStore.ts';
-import { BUTTON_LABELS, HEADER_CSV } from '@/constants/constants.ts';
 import saveAs from 'file-saver';
-import type { TCharacter } from '@/shema/characterShema.ts';
 import { type JSX } from 'react';
+import { useTranslations } from 'next-intl';
+import { createCsv } from '@/actions/download.ts';
 
 const Flyout = (): JSX.Element => {
   const clear = useCardStore((state) => state.clear);
   const selected = useCardStore((state) => state.selected);
```

Comment:
`selected` what? Card? Than use `selectedCard`
Don't afraid to use long meaningfull names